python/lsst/sims/maf/metrics/technicalMetrics.py

```diff
@@ -339,3 +339,41 @@ def run(self, dataSlice, slicePoint=None):
             alpha = np.max([alpha, alphamin])
             alpha = np.min([alphamax, alpha])
         return (r, g, b, alpha)
+
+
+class BruteOSFMetric(BaseMetric):
+    """Assume I can't trust the slewtime or visittime colums.
+    This computes the fraction of time the shutter is open, with no penalty for the first exposure
+    after a long gap (e.g., 1st exposure of the night). Presumably, the telescope will need to focus,
+    so there's not much a scheduler could do to optimize keeping the shutter open after a closure.
+    """
+    def __init__(self, metricName='BruteOSFMetric',
+                 expTimeCol='expTime', mjdCol='observationStartMJD', maxgap=10.,
```

Comment:
Did you mean to have `expTimeCol` = 'visitExposureTime' and not `expTimeCol`='expTime'? With the way it is currently set I had to provide  `expTimeCol`='visitExposureTime' when running the new metric. The old open shutter fraction metric uses `visitExposureTime` as the default so it might be useful to keep this new metric consistent with that behavior.